django/contrib/auth/models.py

```diff
@@ -108,7 +108,7 @@ class Group(models.Model):
     members-only portion of your site, or sending them members-only email
     messages.
     """
-    name = models.CharField(_('name'), max_length=80, unique=True)
+    name = models.CharField(_('name'), max_length=191, unique=True)
```

Comment:
As per comment on the mailing list thread, I think this should be 150 to match altered values in `User` model.

Reply:
I must've had 191 stuck in my head from the discussion. I'll update it now.